notebooks/sam_tests/cmip6_density_transform_pipeline_dev.ipynb

```diff
@@ -8,7 +8,7 @@
     {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.    ddict_nested = match_metrics(ddict_nested_, ddict_nested_, ['areacello', 'thkcello'], exact_attrs = exact_attrs)
</pre><p>I would recommend to load the metric variables into a separate dict here, just to explicitly avoid detrending? That will also get rid of the need to combine table ids! (I now get why that happened before).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jbusecke/cmip6_omz/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>